theano/tensor/basic.py

```diff
@@ -1427,6 +1428,7 @@ class Argmax(Op):
     nout = 1
     E_axis = 'invalid axis'
     __props__ = ()
+    _f16_ok = True
```

Comment:
yes, we just call numpy for the max and argmax. I added tests and numpy handle it well.